lib/config-generator.js

```diff
@@ -331,11 +331,31 @@ ConfigGeneraror.prototype = {
 
         return new Promise(function(resolve, reject) {
             var result = [];
-
+            var isRequire = false;
             jsstana.traverse(ast, function(node) {
-                var match = jsstana.match('(or (call define ?) (call define ? ?) (call define ? ? ?))', node);
+                var match = jsstana.match('(or (call define ?) (call define ? ?) (call define ? ? ?) (call ?.define ?) (call ?.define ? ?) (call ?.define ? ? ?) (call require ?) (call require ? ?) (call require ? ? ?) (call ?.require ?) (call ?.require ? ?) (call ?.require ? ? ?))', node);
```

Comment:
No problem @ipeychev, 

Since this project is "new" to me, please bear with me too.

Yes, the transpiling process generates files like the ones you wrote, but with the `--namespace`, it would also be able to generate files like this

```javascript
// with "--namespace=MyNameSpace"
MyNameSpace.define(["exports", "modal@1.0.0/js/address2.es"], function(exports, foo) {
    foo.log("Hello World!");
});
```

However, if we would run the process a second time, changing the `--namespace`  option's value,  it would change the file above to 

```javascript
// with "--namespace=OtherNameSpace"
OtherNameSpace.define(["exports", "modal@1.0.0/js/address2.es"], function(exports, foo) {
    foo.log("Hello World!");
});
```

I have written a few tests to check this, and it seems to be working fine.

Thanks